test/validation/org/apache/cassandra/bridges/ccmbridge/CCMBridge.java

```diff
@@ -176,4 +180,35 @@ private String executeAndRead(String command, Object... args)
             throw new RuntimeException(e);
         }
     }
+
+    public void captureLogs(String testName)
+    {
+        int count = 1;
+        String clusterLogsPath = ccmDir + "/" + DEFAULT_CLUSTER_NAME;
+        String folderName = testName;
+        String existingFolder = CASSANDRA_DIR + "/build/test/logs/validation/" + folderName;
+        ArchiveClusterLogs capture = new ArchiveClusterLogs();
```

Comment:
All of this class's methods are static, there's no need to instantiate an object to do this.